pkgs/development/libraries/rlottie/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "rlottie";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "Samsung";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256:0fsh3imrl7di9076al1r43crw52q6ryrw642wgdv9rk0fmk3997h";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [
+    "-DLIB_INSTALL_DIR=lib"
```

Comment:
That looks like they should switch to `GNUInstallDirs`, see https://github.com/jtojnar/cmake-snips#creating-custom-install-path-options.

Also the generated `.pc` file will contain incorrect include path with multiple outputs: https://github.com/Samsung/rlottie/blob/e9030d6134b8d35cebc57a0bedcacb0f3942d1fa/CMakeLists.txt#L147

But it looks like there is [Meson](https://github.com/Samsung/rlottie/blob/master/meson.build) support so it might be better to use that over CMake.

Reply:
I need to have a look  at how to write the derivation then. Everything else should be resolved now.

Reply:
Just replacing `cmake` with `meson` and `ninja` and removing `cmakeFlags` should hopefully be enough.

Reply:
Switched to `meson`+`ninja`.